Jenkinsfile

```diff
@@ -65,6 +65,13 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             // Default config
             sh "cp ${params.CONFIG_FILE?.trim()} .config"
 
+            // Add development config
+            echo 'Enabling development config'
+            sh 'echo \'' \
+             + 'CONFIG_DEVEL=y\n' \
+             + 'CONFIG_LOCALMIRROR=\"https://downloads.creatordev.io/pistachio/marduk/dl\"\n' \
```

Comment:
Would rather the server wasn't hammered by developers as it's already under significant load from the CI system. Plus this reduces the delta between us and upstream which I am trying to reduce in the config files.